src/components/molecules/TopNavBar/styled.jsx

```diff
@@ -0,0 +1,20 @@
+import styled, { css } from 'styled-components';
+
+const TopNavBarWarpper = styled.nav`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 13px 16px;
+
+  ${({ theme }) => {
+    return css`
+      border-bottom: 1px solid ${theme.colors.colorDB};
+      h2 {
+        font-size: 18px;
```

Comment:
```suggestion
        font-size: ${theme.fontSizes.xxl};
```
- 해당 부분 폰트사이즈도 theme으로 적용 해 주세요!! 🙇‍♂️

Reply:
앗 다 확인한다고 했는데 빼먹은 부분이 있었네요 🥲 수정 완료하였습니다!